moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -734,4 +748,24 @@ bool JogCalcs::addJointIncrements(sensor_msgs::JointState& output, const Eigen::
 
   return true;
 }
+
+void JogCalcs::removeDimension(Eigen::MatrixXd& jacobian, Eigen::VectorXd& delta_x, const int row_to_remove)
+{
+  Eigen::MatrixXd new_matrix(jacobian.rows() - 1, jacobian.cols());
+  Eigen::VectorXd new_vector(jacobian.rows() - 1);
+  int row_to_fill = 0;
+  for (int orig_matrix_row = 0; orig_matrix_row < jacobian.rows(); ++orig_matrix_row)
```

Comment:
Why didn't you go with the most accepted (and more efficient) answer https://stackoverflow.com/questions/13290395/how-to-remove-a-certain-row-or-column-while-using-eigen-library-c?

Reply:
Side note: I removed the const on the `row_to_remove` argument.

Reply:
done